src/execution/helper.rs

```diff
@@ -50,6 +56,8 @@ pub struct ExecutionHelper {
     // Iter to the read_values array consumed when tx code is executed
     pub execute_code_read_iter: IntoIter<Felt252>,
     pub storage_by_address: StorageByAddress,
+    // Temporary state, until storage_by_address is functional
```

Comment:
I played with `storage_by_address` a bit last week and didn't get anywhere with it. Could you elaborate a bit on what isn't functional?

Reply:
storage_by_address is empty, we need actual data, which right now, for tests, is available in the blockifier data structures.